.ci/appveyor.yml

```diff
@@ -56,6 +56,15 @@ install:
   - "npm config set loglevel warn"
   - "npm install"
 
+  - curl -fsSL ftp://ftp.zlatkovic.com/libxml/iconv-1.9.2.win32.zip -o iconv.zip
```

Comment:
Though it is probably quicker to use `wget "ftp://ftp.zlatkovic.com/libxml/*.win32.zip"` to batch download the files, and `7z` may also have a batch unzip mode.